client/src/main/java/io/confluent/kafka/schemaregistry/ParsedSchema.java

```diff
@@ -42,11 +44,11 @@
   String canonicalString();
 
   /**
-   * Returns the version associated with this schema, if any.
+   * Returns a list of schema references.
    *
-   * @return the version, or null
+   * @return the schema references
    */
-  Integer version();
```

Comment:
why don't we want to return version anymore?

Reply:
Good question.  It's removed from the interface, but some of the implementations still expose this method.   This field is mostly used by Connect.  If someone wants to add a new schema type (and perhaps don't need to make it work with Connect), then they aren't forced to add a version.